plugins/source/gcp/resources/services/resourcemanager/tag_values.go

```diff
@@ -0,0 +1,57 @@
+package resourcemanager
+
+import (
+	resourcemanager "cloud.google.com/go/resourcemanager/apiv3"
+	pb "cloud.google.com/go/resourcemanager/apiv3/resourcemanagerpb"
+	"context"
+	"github.com/apache/arrow/go/v14/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/gcp/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"google.golang.org/api/iterator"
+)
+
+func TagValues() *schema.Table {
```

Comment:
Similar comment about making this function private

Reply:
Made dependent table methods private.